db/schema.rb

```diff
@@ -10,9 +10,52 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2019_12_02_172107) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "loans", force: :cascade do |t|
+    t.date "start_date"
+    t.date "end_date"
+    t.integer "total_price"
+    t.string "status"
+    t.bigint "user_id"
+    t.bigint "offer_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["offer_id"], name: "index_loans_on_offer_id"
+    t.index ["user_id"], name: "index_loans_on_user_id"
+  end
+
+  create_table "offers", force: :cascade do |t|
+    t.string "currency"
+    t.integer "amount"
+    t.integer "interes"
```

Comment:
```suggestion
    t.integer "interest"
```

Reply:
got changed by changing and migrating the migration file above